publications.Rmd

```diff
@@ -15,7 +15,7 @@ pmid_remove <- c("28824762", "26295592", "23288288", "21516278", "18830830",
 ## PubMed IDs to add
 pmid_add <- c("33290556", "11743205", "15761153", "23857251", "26493315", 
               "30002819", "30356428", "31178352", "31857895", "18772890",
-              "18573797", "33300035", "33349709")
+              "18573797", "33300035", "33349709", "34001188")
```

Comment:
Doesn't really matter, but was this needed (it showed up automatically for me)?

Reply:
it was needed for a small time window, but indeed, it's not now. the timing with pubmed is weird. there is a delay to when it's in pubmed (from the publication at GB), and then even if it's there on the website, there is a delay to when `entrez_summary` picks it up .. from earlier this morning, I got this error:
```
> summ <- entrez_summary(db = "pubmed", id = x$ids)
Warning message:
ID 34001188 produced error 'cannot get document summary' 
``` 
.. but now it runs fine.

but anyways, i'll remove this in the next round, which will probably be next week.